w1r3/go/main.go

```diff
@@ -266,7 +266,7 @@ func worker(ctx context.Context, config BenchmarkConfig, tracer trace.Tracer,
 			attribute.String("ssb.version.sdk", getVersion(config, "cloud.google.com/go/storage")),
 			attribute.String("ssb.version.grpc", getVersion(config, "google.golang.org/grpc")),
 			attribute.String("ssb.version.protobuf", getVersion(config, "google.golang.org/protobuf")),
-			attribute.String("ssb.version.http", getVersion(config, "golang.org/x/net")),
+			attribute.String("ssb.version.http-client", getVersion(config, "golang.org/x/net")),
```

Comment:
Go has Http1.1 and http2 support for Storage JSON/XML APIs; Looking over @BrennaEpp benchmark; it uses a helper to select which version of HTTP to use https://github.com/googleapis/google-cloud-go/blob/fbd0576e4591d2aa2121c77bf50ae6ef1b79b572/storage/internal/benchmarks/client_pool.go#L191-L223

Not blocking just calling it out since ssb.version.http makes sense for Go.

Reply:
Agree, created #61